regex-base.cabal

```diff
@@ -90,7 +90,7 @@ library
                , containers >= 0.4 && < 0.7
                , bytestring >= 0.9 && < 0.12
                , array      >= 0.3 && < 0.6
-               , text       >= 1.2.3 && < 1.3
+               , text       >= 1.2.3 && < 1.3 || >=2.0 && <2.1
```

Comment:
Thanks.  Extra nice would be one more line with a comment that justifies the non-continuous version range.